src/managers/EventManager.js

```diff
@@ -0,0 +1,51 @@
+export const getAllEvents = () => {
+    return fetch(`http://localhost:8000/events`, {
```

Comment:
DRY this up and save the baseUrl part of this to a variable, so if you need to change it you can change it in one place. Then, this would look like `${baseUrl}/events`.